creator-node/src/app.js

```diff
@@ -35,10 +35,11 @@ function errorHandler (err, req, res, next) {
 }
 app.use(errorHandler)
 
-const initializeApp = (port, storageDir, s3Bucket, ipfsAPI) => {
+const initializeApp = (port, storageDir, s3Bucket, ipfsAPI, audiusLibs) => {
   app.set('ipfsAPI', ipfsAPI)
   app.set('storagePath', storageDir)
   app.set('redisClient', redisClient)
+  app.set('audiusLibs', audiusLibs)
```

Comment:
@SidSethi i don't know if this is best practice or not compared to having a singleton file. i'm concerned about the file of the libs module but a google search didn't yield much. and i figure this is common memory space so probably doesn't matter 🤷‍♂

Reply:
hmm. lets discuss afk?